peregrine/analysis/print_res.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python
```

Comment:
We previously chose to have acquisition and tracking results saved in Python pickle format. It could be that a more language independent format such as CSV is be a better choice, but we should standardize on one format - a bunch of tools supporting a bunch of formats will make things messy and a maintenance headache. What are your thoughts?